File: iOS/meApp/Features/Settings/MyKids/Strings/MyKidsStrings.swift
//
//  MyKidsStrings.swift
//  meApp
//

import Foundation

struct MyKidsStrings {
    static let title = "My Kids"
    static let subtitle = "Add and manage baby profiles for use with the baby scale."
    static let addABaby = "ADD A BABY"
    static let addBaby = "Add Baby"
    static let save = "SAVE"
    static let saveFailed = "Unable to save baby profile. Please try again."

    /// Field labels + fallbacks for the expandable baby row's expanded detail section (MOB-1605).
    struct Details {
        static let birthday = "Birthday"
        static let biologicalSex = "Biological sex"
        static let birthLength = "Birth length"
        static let birthWeight = "Birth weight"
        /// Shown when a profile field has no recorded value.
        static let empty = "--"
        /// VoiceOver labels for the expand/collapse chevron.
        static let expandAccessibilityLabel = "Show baby details"
        static let collapseAccessibilityLabel = "Hide baby details"
    }

    struct RemoveBaby {
        static let title = "Remove Baby"
        static let message = "Are you sure you want to remove this baby? You will not be able to get it back."
        static let delete = "DELETE"
        static let cancel = "CANCEL"
    }
}

File: Android/app/src/main/java/com/dmdbrands/gurus/weight/features/myKids/strings/MyKidsStrings.kt
package com.dmdbrands.gurus.weight.features.myKids.strings

object MyKidsStrings {
    const val Title = "My Kids"
    const val EmptyDescription = "Add and manage baby profiles for use with the baby scale."
    const val AddABaby = "ADD A BABY"
    const val DeleteBaby = "Delete baby"
    const val EditBaby = "Edit baby"
    const val BabyFallbackPrefix = "Baby"
    const val AvatarFallback = "?"

    // Expanded baby-profile detail rows (see Figma node 34059-32734).
    const val Birthday = "Birthday"
    const val BiologicalSex = "Biological sex"
    const val BirthLength = "Birth length"
    const val BirthWeight = "Birth weight"

    /** Placeholder shown when a baby profile detail is not set. */
    const val ValueUnset = "--"

    // region Accessibility (TalkBack)
    /** Spoken label for the icon-only close button in the app bar. */
    const val accCloseLabel = "Close"

    /** Spoken label for the chevron that expands a baby's profile details. */
    const val accExpandLabel = "Show baby details"

    /** Spoken label for the chevron that collapses a baby's profile details. */
    const val accCollapseLabel = "Hide baby details"
    // endregion
}
